### DISCUSSION TITLE
how to get Apollo upto 10.7.0

### DISCUSSION THREAD

**Discussion by chrishj59** [primefaces/community#2280] [PrimeReact Templates] [answered]
Hi,
I am looking to get Apollo up to the current version as from the download page it is on 10.2.1.
Looking at the Wki [build from source](HTTP://github.com/primefaces/primereact/wiki/Building-From-Source)  there are a number of themes but not the Apollo one.

Thanks

**Comment by melloware** [comment, marked as answer]
you want this guide right? https://github.com/primefaces/primereact/wiki/Migration-Guide-for-Templates
